### ISSUE TITLE
Next.js: Named export 'useDropzone' not found error

### ISSUE THREAD

**Issue by sunadoi** [mantinedev/mantine#4920]
### What package has an issue

@mantine/dropzone

### Describe the bug

I just updated to v7 on my Next.js project (pages router) and came across the following error on a component using dropzone:

```
import { useDropzone } from 'react-dropzone';
         ^^^^^^^^^^^
SyntaxError: Named export 'useDropzone' not found. The requested module 'react-dropzone' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'react-dropzone';
const { useDropzone } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:131:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:213:5)
    at async ModuleLoader.import (node:internal/modules/esm/loader:308:24)
    at async importModuleDynamicallyWrapper (node:internal/vm/module:428:15) {
  page: '/'
}
```

See repro.
Simply added a dropzone component from [next-pages-template](https://github.com/mantinedev/next-pages-template)

### What version of @mantine/* packages do you have in package.json? (Note that all @mantine/* packages must have the same version in order to work correctly)

7.1.0

### If possible, please include a link to a codesandbox with the reproduced problem

https://codesandbox.io/p/sandbox/wandering-resonance-wjrrcp?file=/pages/_app.tsx:24,1

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by rtivital** [comment]
It is a known issue with react-dropzone – https://github.com/react-dropzone/react-dropzone/issues/1259
Nothing can be done on our side until react-dropzone is updated

**Comment by sunadoi** [comment]
Thanks for migrating to the esm package!
Looking forward to this being released!

**Comment by rtivital** [comment]
Fixed in [7.1.1](https://github.com/mantinedev/mantine/releases/tag/7.1.1)
